devices/qemu-arm64

```diff
@@ -0,0 +1 @@
+qemu_arm64
```

Comment:
do you want to change the file name with _
devices/qemu_arm64 ?

Reply:
same here its a symlink